src/components/fire.js

```diff
@@ -0,0 +1,15 @@
+import firebase from 'firebase';
+
+const config = {
+    apiKey: "AIzaSyBrAVy-BKI_8IdPBDBPUbibDlb96fLIRzs",
```

Comment:
If this is linked to your personal google account I wouldn't include this in commit

Reply:
I created a new account for this project thats why its fine